docs/articles/Cookbook.md

```diff
@@ -3,33 +3,90 @@
 ## IO Examples
 ### Padding
 ```csharp
+    // Add 0s until position is divisible by 4
+    filesWriter.WritePadding(0x00, 0x4);
+    // Add 0s until position is divisible by 10
```

Comment:
Let's put the `10` in hex: by `0x10` or by 16.

Reply:
Fixed, thanks!